packages/cli/src/commands/mock.ts

```diff
@@ -0,0 +1,34 @@
+import { Command, flags as oflags } from '@oclif/command';
+import { createServer } from '../util/createServer';
+
+export default class Server extends Command {
```

Comment:
Oh, I forgot! According to our [conversation on Slack](https://stoplight-internal.slack.com/archives/CD6HDERDX/p1554458389110500) this should not be called `server`, but `forward`.

Reply:
Did you considered proxy ?

Reply:
`proxy` works too I think. @philsturgeon ?

Reply:
Sounds good to me!